### ISSUE TITLE
website: security warnings over https

### ISSUE THREAD

**Issue by makc** [MidnightCommander/mc-old#52]
![screen shot 2015-02-16 at 12 04 18](https://cloud.githubusercontent.com/assets/242577/6210867/4d24760c-b5d4-11e4-96e1-849d612ba8ea.png)
I believe this is about following images:

``` html
<img border="0" src="http://www.transifex.net/projects/p/mc/resource/mcpot-4_7_5/chart/image_png" />
<img border="0" src="http://sw.transifex.net/3/static/charts/images/tx-logo-micro.png" />
```

**Comment by makc** [comment]
in fact both URLs are actually dead

**Comment by zyv** [comment]
Fixed by @aborodin on 02.26.2015 09:25:51.
